Block/Configuration.php

```diff
@@ -203,6 +187,10 @@ public function getOrder()
                 "ordertotal" => $lastOrder['grand_total']
             ];
 
+            if (empty($order['userid'])) {
+                $order['email'] = $lastOrder['customer_email'];
```

Comment:
only 1 or the other? It's valid to send the email address over as well. We can use this for campaign emails. Feeds into bigger question of whether the site wants to send us email addresses.

Reply:
This was added to fix a scenario with guest users order tracking failing (as no email address was being sent at all), I can just make it default to always send if that's better?

Reply:
The system will use the userid if present to identify the user and fallback to email if not present. BUT, if both present it will store the email address against the user for use in email campaigns etc.

So I think you should have empty checks for userid and email, and set either if present. If neither is present, then it would fail as we have nothing to identify the user by

Reply:
sorted :)